### ISSUE TITLE
Allow activation of slow mode load gap via keyboard in web UI

### ISSUE THREAD

**Issue by cpsdqs** [mastodon/mastodon#13200]
### Pitch
One of the following:
- pressing the <kbd>↑</kbd> or <kbd>K</kbd> key while focused on the topmost status of a column should move focus to the “n new items” button so it can be activated
- pressing some special keyboard shortcut should activate the “n new items” button and populate the timeline with new statuses (this does not appear to exist, or is not documented in the web UI, in which case this would be a bug report for that instead)

#### Why do you think this feature is needed?
This feature would improve the accessibility of the web UI.

Side note: while it *is* possible to select the topmost status and then press <kbd>Shift</kbd>+<kbd>Tab</kbd> twice to select the “n new items” button, there is no visual feedback for this action at all (missing focus rings have already been reported as an issue it seems), and it’s altogether very awkward compared to usual keyboard navigation in other areas of the web UI.

#### Who would benefit from it?
Users who use both keyboards and slow mode

**Comment by rugk** [comment]
I agree that is really annoying, not only accessibility-wise, but also when you want to navigate with keyboard keys. It just stops navigation when you reach. Apart from that navigating toots works already great IMHO. (/me is not a user using accessibility features due to visual constraints or so)

**Comment by diondiondion** [comment]
> pressing the ↑ or K key while focused on the topmost status of a column should move focus to the “n new items” button so it can be activated

This was implemented in #35455 

We will also implement a keyboard shortcut to focus the "load more" button (tracked via #35316)
